lib/geordi/commands/dump.rb

```diff
@@ -8,15 +8,26 @@
 `geordi dump staging` (with a Capistrano deploy target) remotely dumps the
 specified target's database and downloads it to `tmp/`.
 
+If you are using multiple databases per environment, pass the database name like this:
+
+    geordi dump -d primary
+
+Loading a dump into one of multiple local databases is not supported yet.
+
+When called with the `--load` option, sources the specified dump into the
+development database.
+
 `geordi dump staging -l` (with a Capistrano deploy target and the `--load`
 option) sources the dump into the development database after downloading it.
```

Comment:
Please remove the "When called with..." sentence as discussed, and please move L20+21 ("geordi dump staging -l (with a Capistrano deploy target ...") below "geordi dump staging", atop your insertion. I'd like your addition at the bottom of `DESC`.